src/Stratis.Bitcoin.Features.RPC/RpcSettings.cs

```diff
@@ -32,74 +32,118 @@ public class RpcSettings
         public int RPCPort { get; set; }
 
         /// <summary>Default bindings from config.</summary>
-        public List<IPEndPoint> DefaultBindings { get; set; }
+        public List<IPEndPoint> DefaultBindings => DefaultBinding(this.AllowIp.Any(), this.RPCPort);
 
         /// <summary>List of network endpoints that the node will listen and provide RPC on.</summary>
         public List<IPEndPoint> Bind { get; set; }
 
         /// <summary>List of IP addresses that are allowed to connect to RPC interfaces.</summary>
         public List<IPAddressBlock> AllowIp { get; set; }
 
+        // TODO: Refactor static PrintHelp method.
+        private static RpcSettings settingsHelp;
```

Comment:
Why is the variable called `settingsHelp` when its a `RpcSettings` instance?

Reply:
It's settings defaults intended specifically for use with the `PrintHelp` method.